srv/salt/ceph/dashboard/default.sls

```diff
@@ -70,7 +70,7 @@ dashboard user exists:
 set username and password:
   cmd.run:
     # This command is printed although the 'onchange' statement evaluates as true. This might cause confusion.
-    - name: ceph dashboard ac-user-create {{ dashboard_user }} {{ dashboard_pw }} administrator
+    - name: "echo {{ dashboard_pw }} | ceph dashboard ac-user-create {{ dashboard_user }} -i - administrator"
```

Comment:
we're sure the password doesn't end with a newline? relates to https://github.com/ceph/ceph/pull/39438 @smithfarm this was part of the latest nautilus release, right?

Reply:
@sebastian-philipp That backport PR points to the backport tracker issue which has that information.

Reply:
In that context, the password DeepSea generates is a random string of crap which won't include newlines.

Reply:
(it's just `echo` without `-n` adds a newline)

Reply:
> (it's just `echo` without `-n` adds a newline)

Good point!

```
$ echo "foo" | hexdump -c
0000000   f   o   o  \n                                                
0000004
```